### ISSUE TITLE
Social image error in linux

### ISSUE THREAD

**Issue by condef5** [sergiodxa/contentz#23]
**Describe the bug**
Build image failed in linux
**Additional context**
This is the error that shows:
```sh
Google Chrome was not found, it's required to generate the social images.
```

**Comment by sergiodxa** [comment]
Contentz use puppeteer but to reduce install size it uses puppeteer-core which comes without Chrome, so you have Google Chrome installed?

**Comment by condef5** [comment]
Yes, I have Google Chrome, but in linux my chrome path is `'/usr/bin/google-chrome'`

**Comment by sergiodxa** [comment]
This should be fixed in v3.2.3, could you update and try again?
